sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -286,11 +287,56 @@ def test_predict_pred_interval(Forecaster, fh, alpha):
             with pytest.raises(NotImplementedError, match="prediction intervals"):
                 f.predict(return_pred_int=True, alpha=alpha)
 
+def _check_predict_quantiles(pred_quantiles: list, y_train: pd.Series, fh, alpha):
+    # alpha can be a float or a list
+    # if a list, it has to be sorted
+    # check if alpha is increasing and in the given interval
+    if isinstance(alpha, list):
```

Comment:
hm, this looks more like a test that the `BaseForecaster` should carry out? `alpha` is an input, and we know that the `alpha` in the test are fine? 

I suggest:
* move this to the `BaseForecaster` as an input check for the `alpha`; extend this that it checks that `alpha` is a `float`, or list of `float`.
* add a test which checks that `BaseForecaster` raises an error for "wrong" `alpha`.

Reply:
isn't there already a `check_alpha`, btw?